assets/dev/scss/frontend/_container.scss

```diff
@@ -34,6 +34,8 @@
 	--overlay-mix-blend-mode: initial;
 	--overlay-opacity: 1;
 	--overlay-transition: 0.3s;
+	--e-con-grid-template-columns: repeat(1, 1fr);
+	--e-con-grid-template-rows: repeat(1, 1fr);
```

Comment:
Not sure that this is related to this task.

Reply:
@davseve The problem was that there was no initial or fallback value for the column and row control, which caused an empty 'empty view' when the user removed the column control value.

![Screenshot 2023-03-30 at 10 02 07](https://user-images.githubusercontent.com/94352322/228770292-4aa37a6b-dbf7-45dc-a678-52e19b78d825.png)

I have added it, so that the styling doesn't break when the user removes the values, which was partially what happened in the video.